backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java

```diff
@@ -441,18 +440,30 @@ public void getHydratedExperimentRunsInProject(
     Set<String> experimentIdSet = new HashSet<>();
     Set<String> vertaIdList = new HashSet<>();
     Set<String> projectIdSet = new HashSet<>();
+    Set<String> experimentRunIdSet = new HashSet<>();
     for (ExperimentRun experimentRun : experimentRuns) {
       vertaIdList.add(experimentRun.getOwner());
       experimentIdSet.add(experimentRun.getExperimentId());
       projectIdSet.add(experimentRun.getProjectId());
+      experimentRunIdSet.add(experimentRun.getId());
     }
 
-    Map<String, Actions> actions = new HashMap<>();
+    Map<String, Actions> projectActions = new HashMap<>();
     if (projectIdSet.size() > 0) {
-      actions =
+      projectActions =
           roleService.getSelfAllowedActionsBatch(
               new ArrayList<>(projectIdSet), ModelDBServiceResourceTypes.PROJECT);
     }
+    List<Action> projectActionList =
```

Comment:
thoughts on why this change shows up?

Reply:
I think it was a part of some other branch that I was basing my changes on before, but not in master now. Sorry for that